eng/Version.Details.xml

```diff
@@ -12,6 +12,10 @@
       <Uri>https://github.com/dotnet/emsdk</Uri>
       <Sha>9f2345b3c5f43dbf34790e21657ae1f2445cd06a</Sha>
     </Dependency>
+    <Dependency Name="System.ServiceModel.Primitives" Version="4.9.0-rc2.21473.1">
```

Comment:
@mmitche just checking, do we want this product dependency in the graph?

Reply:
Yep it's fine.